base/lib/core/presenter/AboutDialog.cpp

```diff
@@ -0,0 +1,182 @@
+                              background-color: rgb(18,23,26);
+                              margin:0px;
+                              }
+                              /* QListView::item{
+                              color: rgb(0,146,207);
+                              }*/
+                              QScrollBar::right-arrow:horizontal, QScrollBar::left-arrow:horizontal
+                              {
+                              border: none;
+                              background: none;
+                              color: none;
+                              }
+                              QScrollBar::add-line:vertical {
+                              border: none;
+                              background: none;
+                              }
+                              QScrollBar::sub-line:vertical {
+                              border: none;
+                              background: none;
+                              }
+                              )_");
+  for(auto item : map)
+  {
+    softwareList->addItem(item.first);
+  }
+
+
+  // license
+  auto license = new QPlainTextEdit{this};
+  license->move(280,230);
+  license->resize(185,183);
+  license->setFont(m_catamaranFont);
+  license->setReadOnly(true);
+  connect(softwareList,&QListWidget::currentTextChanged,this,
+          [=] (const QString& currentText)
+  {
+    license->setPlainText(map.at(currentText));
+  });
+}
+
+void AboutDialog::mousePressEvent(QMouseEvent *event)
+{
+
+  QPointF pos = event->localPos();
+  if(m_mouseAreaLabri.contains(pos))
+  {
+    QDesktopServices::openUrl(QUrl("https://www.labri.fr/"));
+  }
+  else if(m_mouseAreaBlueYeti.contains(pos))
+  {
+    QDesktopServices::openUrl(QUrl("http://www.blueyeti.fr/"));
+  }
+  else if(m_mouseAreaScrime.contains(pos))
+  {
+    QDesktopServices::openUrl(QUrl("https://scrime.u-bordeaux.fr/"));
+  }
+  this->close();
+}
+
+void AboutDialog::mouseMoveEvent(QMouseEvent *event)
+{
+  QPointF pos = event->localPos();
+  if( m_mouseAreaLabri.contains(pos)
+      || m_mouseAreaBlueYeti.contains(pos)
+      || m_mouseAreaScrime.contains(pos))
+  {
+    this->setCursor(Qt::PointingHandCursor);
+  }
+  else
+  {
+    this->setCursor(Qt::ArrowCursor);
+  }
+}
+void AboutDialog::paintEvent(QPaintEvent *event)
+{
+  QPen textPen(QColor("#0092cf"));
+  QPen titleText(QColor("#aaaaaa"));
+  QPen rectPen(QColor("#03c3dd"));
+  QBrush rectBrush(QColor(18,23,26));
+
+  auto version_text = QStringLiteral("Version: %1.%2.%3-%4 '%5'\n")
+                      .arg(SCORE_VERSION_MAJOR)
+                      .arg(SCORE_VERSION_MINOR)
+                      .arg(SCORE_VERSION_PATCH)
+                      .arg(SCORE_VERSION_EXTRA)
+                      .arg(SCORE_CODENAME);
+
+  QString commit{GIT_COMMIT};
+
+  if (!commit.isEmpty())
+  {
+    version_text += tr("Commit: %1\n").arg(commit);
+  }
+
+  auto copyright_text = QString("Copyright © Ossia "+ QString::number(QDate::currentDate().year()));
```

Comment:
@OSSIA/core is this the correct form for the copyright ?

Reply:
well I don't really know what we should put here
and also is the license really copyright, or is it some kind of Creative Commons or whatever?